openml-lightgbm-module/openml-lightgbm/pom.xml

```diff
@@ -94,66 +95,23 @@
             <artifactId>commons-csv</artifactId>
             <scope>test</scope>
         </dependency>
-
     </dependencies>
 
     <build>
-
         <resources>
             <resource>
                 <directory>${project.basedir}/target/resources</directory>
             </resource>
         </resources>
 
         <plugins>
-
-            <!-- Copy the LightGBM shared libraries to the resources folder -->
-            <plugin>
```

Comment:
What replaces this functionality?

the new assembly/dist.xml? But does it know where to unpack all the libs and .so?

Reply:
The *.so and *.so.1.0.0 are generated, and moved as `resources`
Regarding unpack, now we are only unpacking lightgbmlib.jar using `maven-antrun-plugin`